src/main/common/utils/numericUtils.ts

```diff
@@ -14,3 +14,17 @@ export function toNumberOrUndefined (value: any): number {
 
   return isNaN(numberVal) ? undefined : numberVal
 }
+
+export function containsAThousandSeparator (input: string): boolean {
+  const output: string[] = input.split(',')
+
+  for (let i: number = 0; i < output.length; i++) {
+    if (i === output.length - 1 && output[i].length < 3) {
+      return false
+    } else if (output[i].length > 3) {
+      return false
+    }
+  }
+
+  return true
+}
```

Comment:
What do you think about using a regular expression to check this?
```
export function usesThousandsSeparators (input: string): boolean {
  return input.match(/^[1-9]\d{0,2}(\,\d{3})+(\.\d*)?$/)
}
```
It's a little more concise.

Reply:
better